README.md

```diff
@@ -130,7 +161,12 @@ spec:
     priority: 4
     match:
     - nodeLabel: "node-role.kubernetes.io/worker"
+  highAvailability:
+    enableHA: true
+    preferredPrimary: "ens7f0"
```

Comment:
I still think HA config should  be outside ptpconfig , ptpConfig is per nic profile , so HA config new API  introduced , which  is read by linuxPtpdaemon to configure HA.
@josephdrichard

Reply:
also this requires to update clientset for ptp-opertaor